src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -34,7 +34,11 @@ public CommandResult execute(Model model) {
                 String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, model.getFilteredPersonList().size()));
     }
 
-    @Override
```

Comment:
Also, is it neccessary for the equals method to have JavaDocs, since the overridden Object superclass already has JavaDocs? 😺

Reply:
Yeah I also thought it wasn't necessary, but the checkstyle test pointed it out as an error so I had to add it here 🥲🥲🥲. This is also why I had to have a PR with both the update dg and checkstyle commit so that it can pass the tests